packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -80,7 +80,12 @@ class FlutterCommandRunner extends CommandRunner<void> {
         help: 'Suppress analytics reporting for the current CLI invocation.');
     argParser.addFlag('disable-telemetry',
         negatable: false,
-        help: 'Disable telemetry reporting when this command runs.');
+        help: 'Disable telemetry reporting each time a Flutter or Dart '
```

Comment:
Are the `flutter` and `dart` CLI commands the only  things affected? Or will it also disable telemetry if I run `gen_snapshot`, for instance?

Basically, can I infer from that statement that all executables shipped with Dart or Flutter will stop reporting telemetry if I set this flag?

I think the answer is yes, in which case this looks correct, but if it's not, then you might want to use specific references to the (lowercase) `flutter` and `dart` executables instead of using the proper names for the SDKs (or maybe we should just make that the way it works, and keep the proper names).

Reply:
You are correct, eventually all tools will have this package and once we disable one, every tool using `package:unified_analytics` will have reporting disabled. And makes sense, I will lowercase the letters now